### ISSUE TITLE
SlevomatCodingStandard.Classes.ClassMemberSpacing not working well with PHP 8.0 attributes

### ISSUE THREAD

**Issue by radek-bruha** [slevomat/coding-standard#1155]
Hello, there is a problem that ClassMemberSpacing sniff not allowing PHP 8.0 attributes between method and it's comment.

```php
slevomat/coding-standard   dev-master 47cf967 
squizlabs/php_codesniffer  dev-master 285de0d 
```

```php
  /**
   * @var DateTimeImmutable
   */
  private DateTimeImmutable $date;

  /**
   * @return DateTimeImmutable
   */
  #[GraphQL\Field] // This is problem :(
  public function getDate(): DateTimeImmutable
  {
      return $this->date;
  }
```

```
----------------------------------------------------------------------
 36 | ERROR | [x] Expected 1 blank line between class members, found 5.
    |       |     (SlevomatCodingStandard.Classes.ClassMemberSpacing.IncorrectCountOfBlankLinesBetweenMembers)
----------------------------------------------------------------------
```

Without `#[GraphQL\Field]` line everything works well.

**Comment by Ilyes512** [comment]
I also have a problem with the same sniffs in combination with PHP 8 property promotions (https://php.watch/versions/8.0/constructor-property-promotion).

It will suggest adding all kinds of new lines where they do not belong:
```
    ---------- begin diff ----------
--- Original
+++ New
@@ -19,9 +19,17 @@
         //
     }

+
+
+
+
     public function create(ResponseInterface $response): Collection
     {
         $this->validateResponse($response);
+
+
+
+

         return Collection::make();
     }
    ----------- end diff -----------
    ```

**Comment by kukulich** [comment]
@Ilyes512 Have you tested `dev-master` ?

**Comment by Ilyes512** [comment]
@kukulich Ok... I made sure the only issue I had was the one above... after setting this package to `dev-master` and updating, it was not reporting any issues! :) Nice!

**Comment by kukulich** [comment]
Waiting for https://github.com/squizlabs/PHP_CodeSniffer/pull/3203

**Comment by dakorpar** [comment]
it's now merged :tada:

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/f8d8f226d2a0fe9a16f801229f5d396c093c75a4
